cmd/list/breakglasscredential/cmd.go

```diff
@@ -0,0 +1,81 @@
+package breakglasscredential
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/rosa/pkg/input"
+	"github.com/openshift/rosa/pkg/ocm"
+	"github.com/openshift/rosa/pkg/output"
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+var Cmd = &cobra.Command{
+	Use:     "break-glass-credentials",
+	Aliases: []string{"break-glass-credential", "breakglasscredential", "breakglasscredentials"},
+	Short:   "List break glass credential",
+	Long:    "List break glass credential for a cluster.",
+	Example: `  # List all break glass credentials for a cluster named 'mycluster'"
+  rosa list break-glass-credentials -c mycluster`,
+	Run:    run,
+	Args:   cobra.NoArgs,
+	Hidden: true,
+}
+
+func init() {
+	ocm.AddClusterFlag(Cmd)
+	output.AddFlag(Cmd)
+}
+
+func run(cmd *cobra.Command, _ []string) {
+	r := rosa.NewRuntime().WithAWS().WithOCM()
+	defer r.Cleanup()
+	err := runWithRuntime(r, cmd)
+	if err != nil {
+		r.Reporter.Errorf(err.Error())
+		os.Exit(1)
+	}
+}
+
+func runWithRuntime(r *rosa.Runtime, cmd *cobra.Command) error {
+	clusterKey := r.GetClusterKey()
+	cluster := r.FetchCluster()
+
+	input.CheckIfHypershiftClusterOrExit(r, cluster)
+
+	// Load any break glass credential for this cluster
+	r.Reporter.Debugf("Loading break glass credentials for cluster '%s'", clusterKey)
+	breakGlassCredentials, err := r.OCMClient.GetBreakGlassCredentials(cluster.ID())
+	if err != nil {
+		return fmt.Errorf("failed to get break glass credentials for cluster '%s': %v", cluster.ID(), err)
+	}
+
+	if output.HasFlag() {
+		err = output.Print(breakGlassCredentials)
+		if err != nil {
+			return fmt.Errorf("%v", err)
+		}
+	}
+
+	if len(breakGlassCredentials) == 0 {
+		return fmt.Errorf("there are no break glass credentials for this cluster")
```

Comment:
Info or Warning usually

Reply:
Hmmm this one is a bit tricky. It's inside runWithRuntime which only returns error.

Reply:
```
r.Reporter.Infof("There are no break glass credentials for this cluster.")
return nil
```
I was thinking something like this

Reply:
Ah I'll try that, thanks @andreadecorte